packages/react-composites/src/composites/CallComposite/components/LocalPreview.tsx

```diff
@@ -120,6 +122,11 @@ export const LocalPreview = (props: LocalPreviewProps): JSX.Element => {
               disabled={!microphonePermissionGranted || !cameraPermissionGranted}
               showLabel={true}
               styles={devicesButtonStyles}
+              strings={
```

Comment:
Consider updating the icon here too to be the speaker instead of the microphone like you did in the more drawer

Reply:
Had to dive into the DevicesButton and make changes there. There is no prop that can passed down to change the icons unfortunately (maybe this is something we can add into the component for the future).